sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -0,0 +1,334 @@
+        :type sequence_numbers: int or list[int]
+        :rtype: None
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sync_samples/sample_code_servicebus.py
+                :start-after: [START cancel_scheduled_messages]
+                :end-before: [END cancel_scheduled_messages]
+                :language: python
+                :dedent: 4
+                :caption: Cancelling messages scheduled to be sent in future
+        """
+        self._open()
+        if isinstance(sequence_numbers, int):
+            numbers = [types.AMQPLong(sequence_numbers)]
+        else:
+            numbers = [types.AMQPLong(s) for s in sequence_numbers]
+        request_body = {'sequence-numbers': types.AMQPArray(numbers)}
+        return self._mgmt_request_response_with_retry(
+            REQUEST_RESPONSE_CANCEL_SCHEDULED_MESSAGE_OPERATION,
+            request_body,
+            mgmt_handlers.default
+        )
+
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str,
+        **kwargs
+    ):
+        # type: (str, Any) -> ServiceBusSender
+        """Create a ServiceBusSender from a connection string.
+
+        :param conn_str: The connection string of a Service Bus.
+        :keyword str queue_name: The path of specific Service Bus Queue the client connects to.
+        :keyword str topic_name: The path of specific Service Bus Topic the client connects to.
+        :keyword bool logging_enable: Whether to output network trace logs to the logger. Default is `False`.
+        :keyword int retry_total: The total number of attempts to redo a failed operation when an error occurs.
+         Default value is 3.
+        :keyword transport_type: The type of transport protocol that will be used for communicating with
+         the Service Bus service. Default is `TransportType.Amqp`.
+        :paramtype transport_type: ~azure.servicebus.TransportType
+        :keyword dict http_proxy: HTTP proxy settings. This must be a dictionary with the following
+         keys: `'proxy_hostname'` (str value) and `'proxy_port'` (int value).
+         Additionally the following keys may also be present: `'username', 'password'`.
+        :rtype: ~azure.servicebus.ServiceBusSenderClient
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sync_samples/sample_code_servicebus.py
+                :start-after: [START create_servicebus_sender_from_conn_str_sync]
+                :end-before: [END create_servicebus_sender_from_conn_str_sync]
+                :language: python
+                :dedent: 4
+                :caption: Create a new instance of the ServiceBusSender from connection string.
+
+        """
+        constructor_args = cls._from_connection_string(
+            conn_str,
+            **kwargs
+        )
+        return cls(**constructor_args)
+
+    def send(self, message, message_timeout=None):
```

Comment:
We should add `**kwargs` and consider moving `message_timeout` as a kwarg.
Also wondering if this should be `send_timeout`? What are other languages using?

Reply:
interesting, other languages don't have "timeout" in send.
At least I know that JS is using timeout provided by retry_options.

I'm leaning towards to removing this parameter now for the sake of consistency with other languages.

Reply:
updated to "timeout" first, pending discussion for preview2

Reply:
moved the topic to here: https://github.com/Azure/azure-sdk-for-python/issues/10602